cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,122 +66,87 @@ describe("User Creation", () => {
   });
 
   it("Update the existing user profile and verify its reflection", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.verifyElementContainsText(
-      "username-profile-details",
-      "devdistrictadmin",
-    );
-    userCreationPage.clickElementById("edit-cancel-profile-button");
-    userCreationPage.typeIntoElementByIdPostClear(
-      "firstName",
-      "District Editted",
-    );
-    userCreationPage.typeIntoElementByIdPostClear("lastName", "Cypress");
-    userCreationPage.selectDropdownOption("gender", "Male");
-    userCreationPage.typeIntoElementByIdPostClear("phoneNumber", phone_number);
-    userCreationPage.typeIntoElementByIdPostClear(
-      "altPhoneNumber",
-      emergency_phone_number,
-    );
-    userCreationPage.typeIntoElementByIdPostClear("email", "test@test.com");
-    userCreationPage.typeIntoElementByIdPostClear("weekly_working_hours", "14");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    cy.verifyContentPresence("#username-profile-details", ["devdistrictadmin"]);
+
+    userCreationPage.clickEditCancelProfileButton();
+    userCreationPage.typeIntoFirstNameAndClear();
+    userCreationPage.typeIntoLastNameAndClear();
```

Comment:
```suggestion
    userCreationPage.typeLastName(lastName);
```

- make all the similar changes i mention for firstName in this function as well

Reply:
![{598225F2-79B7-4202-B3B1-48EEE28DFACE}](https://github.com/user-attachments/assets/f533e0d1-1863-4d83-9931-2aed774f91ea)